scripts/qemu-check.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Commit message says 'add set -e' the body of the message says that we add ignore for time out. This should really be two separate patches.

Reply:
The commit message is probably a bit too short, I meant: ignore set -e when calling qemu because qemu always times out.